dist/mainnet.addresses.json

```diff
@@ -55,7 +56,10 @@
         "0xC9AD279994980F8DF348b526901006972509677F",
         "0x802d0f2f4b5f1fb5BfC9b2040a703c1464e1D4CB",
         "0x418De6227499181B045CAdf554030722E460881a",
-        "0x09b1b33BaD0e87454ff05696b1151BFbD208a43F",
-        "0x1D310a6238e11c8BE91D83193C88A99eB66279bE"
+        "0x94C476E4675FAaC1Bf60211c5E39486bCF720cBD",
```

Comment:
`0x...cBD` is wrong. This should be `0x09b1b33BaD0e87454ff05696b1151BFbD208a43F`